### PR TITLE
Add `function.prototype.name` shim

### PR DESCRIPTION
**Pull request by raejin**
As title.

Please do not merge until https://github.com/ljharb/function.prototype.name/pull/13 gets resolved. Since the shim() doesn't work in IEs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "airbnb-browser-shims",
-  "version": "1.11.1",
+  "version": "1.12.0",
```
Comment (ljharb) [inline package.json]: This should be reverted; version numbers should never be bumped in PRs.

**Review by ljharb — CHANGES_REQUESTED** [review]
LGTM pending https://github.com/ljharb/function.prototype.name/pull/13 and this change

**Comment by raejin** [comment]
@ljharb Good to go!

File: browser-only.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ require('webrtc-adapter');
 
 require('whatwg-fetch');
 
+require('function.prototype.name').shim();
```
Comment (ljharb) [inline browser-only.js:18]: i feel like maybe this would be better in js-shims, but in an IE 9-11 target. Let's chat about it tomorrow.

Comment (raejin) [reply browser-only.js:18]: Sounds good to me. I agree it is more of a js-shim.

**Comment by raejin** [comment]
After some reflection, @ljharb:

Since this is a IE specific issue, regardless of pre or post ECMAScript5, should we shim this in browser shims instead?

**Comment by ljharb** [comment]
Since this is a language feature, and there's no way to target IE 10 or 11 (since they lack conditional comments), this has been added to `airbnb-js-shims` v1.2.0.

**Pull request closed without merging**